### PR TITLE
Allow theming of mui components

### PR DESCRIPTION
**Pull request by fnberta**

### CODE REVIEW

File: src/Admin.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ import adminReducer from './reducer';
 import crudSaga from './sideEffect/saga';
 import CrudRoute from './CrudRoute';
 import Layout from './mui/layout/Layout';
-import withAppTitle from './mui/layout/withAppTitle';
+import withAppTitleAndTheme from './mui/layout/withAppTitleAndTheme';
 
-const Admin = ({ restClient, dashboard, children, title = 'Admin on REST', appLayout = withAppTitle(title)(Layout) }) => {
+const Admin = ({ restClient, dashboard, children, title = 'Admin on REST', theme = {}, appLayout = withAppTitleAndTheme(title, theme)(Layout) }) => {
```
Comment (djhi) [inline src/Admin.js]: What is the behavior with `{}` as default ?

Comment (fnberta) [inline src/Admin.js]: With `{}` the default theme will be used. `getMuiTheme()` expects an object containing the values you would like to override. Hence passing an empty object, nothing will be overridden.

File: src/Admin.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ import adminReducer from './reducer';
 import crudSaga from './sideEffect/saga';
 import CrudRoute from './CrudRoute';
 import Layout from './mui/layout/Layout';
-import withAppTitle from './mui/layout/withAppTitle';
+import withAppTitleAndTheme from './mui/layout/withAppTitleAndTheme';
```
Comment (fzaninotto) [inline src/Admin.js]: this is starting to look more and more like a generic `withProps` HOC. I suggest you rename it and move it ip the directory structure.

Comment (djhi) [inline src/Admin.js]: https://github.com/acdlite/recompose ?

Comment (fzaninotto) [inline src/Admin.js]: In the future, yes. But for a 3 lines function, it's overkill.

Comment (fnberta) [inline src/Admin.js]: like this?

Comment (fzaninotto) [inline src/Admin.js]: Yep, great!

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**